balena.yml

```diff
@@ -0,0 +1,32 @@
+
+name: meetbot
+type: sw.application
+description: >-
+  Meetbot is a configurable Google Meet bot aimed at reducing friction in video calls.
+post-provisioning: >-
+  ## Usage instructions
+
+  Allow a few minutes for your device to download the application. Once it's
+  completed, the meetbot dashboard should be accessible on the public device URL.
+assets:
+  repository:
+    type: blob.asset
+    data:
+      url: 'https://github.com/balena-io-playground/meetbot'
+  logo:
+    type: blob.asset
+    data:
+      url: 'https://raw.githubusercontent.com/balena-io-playground/meetbot/master/img/logo.png'
+data:
+  applicationEnvironmentVariables:
+    - GOOGLE_PASSWORD=
+    - GOOGLE_TOTP_SECRET=
+    - GOOGLE_EMAIL=
```

Comment:
What's the point of adding these here if they don't have a default value (which makes sense ofc)?

Reply:
@tmigone I added them as prompts for people to look and read about these values. We can't have defaults for these and the people need to go through the "authentication process" in order to provide us these values.

Reply:
I left them blank instead of "ENTER_GOOGLE_PASSWORD_HERE"